specification/src/main/asciidoc/jakarta-concurrency.adoc

```diff
@@ -2810,3 +2810,104 @@ components to manually control global transaction demarcation
 boundaries. Task implementations may optionally begin, commit, and
 roll-back a transaction. See EE.4 for details on transaction management
 in Jakarta EE.
+
+== Scheduled Methods
+
+The `jakarta.enterprise.concurrent.Schedule` annotation can be applied to a
+CDI managed bean method to cause the method to run automatically on a
+schedule. The method must have a `void` return type and no parameters.
```

Comment:
```suggestion
schedule. The method must have a `void` return type and either no parameters or a parameter that observes the jakarta.enterprise.Startup event.
```

Clarification: Doesn't this go against the Javadoc updates?

Reply:
> Clarification: Doesn't this go against the Javadoc updates?

No. There are two different usage patterns discussed in the Javadoc.  One is for `@Schedule` annotating a method.  Another is for `@Schedule` being used by `@Asynchronous runAt=` (what is already possible and not new).  The specification section here is only discussing the former pattern where the requirement for no parameters on the method is correct.  I can see where the word "applied to" here can be misinterpreted, and thanks for helping to spot that.  To help clarify that it is only the first pattern here, I can change the words "annotation can be applied to" to instead be "annotation can annotate"

Reply:
It is updated now.